pkg/columns/columninfo.go

```diff
@@ -43,11 +56,47 @@ type Column[T any] struct {
 	columnType    reflect.Type // cached type info from reflection
 }
 
+func (ci *Column[T]) getWidth(params []string) (int, error) {
+	if len(params) == 1 {
+		return 0, fmt.Errorf("missing %q value for field %q", params[0], ci.Name)
+	}
+	if params[1] == "type" {
```

Comment:
What about setting `MaxWidth` by default to "type" if it is an integer?

Reply:
You mean when defining like so: `column:"intField,width,maxWidth,minWidth"`?

Reply:
No, I meant that if the user doesn't explicitly specify the `maxWtidth` in the `column` tag of an `int` field, then we automatically set `maxWtidth` to the max for that type of int.

Reply:
Hmm, I think it's a good to set it as default, but how would you go back to scaling "naturally" for those fields, then - in case you don't want to use the `type` defaults?

Reply:
Is there any case where someone won't want `types` defaults? I mean, if the user is using auto-scaling, they want to take advantage of all the screen to get those long fields displayed almost completely, I don't think they want to "waste" the space for fields that we are sure will not be longer than the max we are setting by default. Don't you think? The only case I can see that they won't want this default is, maaaybe for "styling" reasons 😅.

Reply:
Hehe, _I_ wouldn't to that as well, but yeah, "styling" reasons. Maybe we should just go with it and re-add it somehow if it's really needed.

Reply:
Anyway, I was thinking that for the IG use-case, we will set the `minWidth=types` for all the `int` fields, but there could be other use cases where it won't be like that. So, I am fine wither way 🙂